scripts/ci/coverage/coverage_analysis.py

```diff
@@ -13,6 +13,13 @@ class Json_report:
         "components":[]
     }
 
+    simulators = [
+        'unit_testing',
+        'native',
+        'quemu',
```

Comment:
hold up, isnt this a typo?

Reply:
https://github.com/zephyrproject-rtos/zephyr/pull/72085

Reply:
Yes, typo issue. Thank you.